images-list

```diff
@@ -220,24 +220,29 @@ quay.io/calico/cni rancher/mirrored-calico-cni v3.13.4
 quay.io/calico/cni rancher/mirrored-calico-cni v3.16.5
 quay.io/calico/cni rancher/mirrored-calico-cni v3.17.2
 quay.io/calico/cni rancher/mirrored-calico-cni v3.18.1
+quay.io/calico/cni rancher/mirrored-calico-cni v3.19.0
 quay.io/calico/ctl rancher/calico-ctl v3.16.5
 quay.io/calico/ctl rancher/calico-ctl v3.16.6
 quay.io/calico/ctl rancher/calico-ctl v3.17.1
 quay.io/calico/ctl rancher/calico-ctl v3.17.2
 quay.io/calico/ctl rancher/calico-ctl v3.18.1
+quay.io/calico/ctl rancher/calico-ctl v3.19.0
```

Comment:
Do we need this since it is already switched to mirrored-?

Reply:
I don't think so, removing